internal/cli/events/organization/list.go

```diff
@@ -53,11 +47,11 @@ func (opts *ListOpts) Run() error {
 	var r *atlas.EventResponse
 	var err error
 
-	if opts.orgID != "" {
-		r, err = opts.store.OrganizationEvents(opts.orgID, listOpts)
-	} else if opts.projectID != "" {
-		r, err = opts.store.ProjectEvents(opts.projectID, listOpts)
+	if err = opts.ValidateOrgID(); err != nil {
```

Comment:
When I tried addressing the lint error (with the suggestion given by the linter), I got this error:
`shadow: declaration of "err" shadows declaration`

Reply:
you have to move 	`var err error` after 
```
	if err := opts.ValidateProjectID(); err != nil {
		return err
	}
```

Reply:
same in the org